sycl/test/basic_tests/id.cpp

```diff
@@ -437,6 +437,36 @@ int main() {
           (number_5 == numValue));
   }
 
+  {
+    cl::sycl::id<1> one_dim_id(64);
+    cl::sycl::id<1> one_dim_id_copy(64);
+    cl::sycl::id<2> two_dim_id(64, 1);
+    cl::sycl::id<2> two_dim_id_copy(64, 1);
+    cl::sycl::id<3> three_dim_id(64, 1, 2);
+    cl::sycl::id<3> three_dim_id_copy(64, 1, 2);
+
+    assert((+one_dim_id) == one_dim_id);
+    assert(-(-one_dim_id) == one_dim_id);
```

Comment:
Can we compare the result with actual numbers? Otherwise even if unary minus did nothing the check would pass.

Reply:
Created an object with negative numbers, now the comparison starts with it